model/data/databricks.py

```diff
@@ -225,14 +225,13 @@ def get_op(self) -> pd.DataFrame:
             .filter(F.col("fyear") == self._year)
             .withColumn("dataset", F.lit("NATIONAL"))
             .withColumn("sitetret", F.lit("NATIONAL"))
+            .withColumn("sushrg_trimmed", F.lit("HRG"))
```

Comment:
worth adding the `imd_quintile` column with a placeholder value?

```suggestion
            .withColumn("sushrg_trimmed", F.lit("HRG"))
            .withColumn("imd_quintile", F.lit(0))
```